docs/snp.md

```diff
@@ -84,6 +84,13 @@ The `--non-upm` option can be specified with the above command if a non-upm vers
 of the kernel is desired. The `setup-host` command must be run with this same option 
 if launching the guest with a non-upm kernel.
 
+A user can launch multiple SNP guests at the same time using unique guest name and guest qemu port. Guest qemu port can be set by the export of HOST_SSH_PORT environment variable.
```

Comment:
Yes, you are right.

Reply:
Follow up question. If I don't pass guest-name parameter when launching a guest, will it default to an specific name or will it fail?

Reply:
In this case, guest-image gets created directly inside the launch directory(no separate guest directory will be created in this case)

If user prefers to create a separate guest working directory, then --guest-name will be helpful to create all files related to guest launch under a separate guest directory inside launch directory

Reply:
Got it. Well, I think maybe adding an optional parameter to launch guest that you can specify the port so that you dont have to modify the script everytime would be helpful

Reply:
Thanks for the idea @DGonzalezVillal !!

Addressed this change via addition of `guest-port` command option along with `guest-name`. Please review and comment any changes required